tests/integration/nmstatectl_test.py

```diff
@@ -108,7 +108,7 @@ def test_show_command_only_non_existing():
     ret = libcmd.exec_cmd(SHOW_CMD + ['non_existing_interface'])
     rc, out, err = ret
 
-    assert_rc(rc, RC_SUCCESS, ret)
+    assert_rc(rc, RC_SUCCESS, exec_cmd_report(ret))
```

Comment:
What is the benefit of calling a method instead of just writing

`assert rc == RC_SUCCESS, exec_cmd_report(ret)`

directly?.

Reply:
You are right. In the process of changing it I misses the outer picture.